### DISCUSSION TITLE
can't figure out how to use ERC20Mock.sol on the stablecoin development

### DISCUSSION THREAD

**Discussion by ighodaro-daniel** [Cyfrin/foundry-full-course-cu#2240] [Q&A]
the  current openzeppelin ERC20Mock.sol has no parameter and quite different on how Patrick engage with it , please does anyone has any hint on how to work with it

**Comment by ighodaro-daniel** [comment]
please any help

**Reply by EngrPips** [reply]
Hello @ighodaro-daniel, Please check here https://github.com/Cyfrin/foundry-full-course-cu/blob/main/chronological-updates.md , Specifically Lesson 12 updates address this.
